src/applications/caregivers/containers/ConfirmationPage.jsx

```diff
@@ -3,13 +3,15 @@ import moment from 'moment';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 import { connect } from 'react-redux';
 
+// import recordEvent from 'platform/monitoring/record-event';
 import { focusElement } from 'platform/utilities/ui';
 import { scrollToTop } from 'applications/claims-status/utils/page';
 
 const ConfirmationPage = props => {
   useEffect(() => {
     focusElement('.schemaform-title > h2');
     scrollToTop();
+    // recordEvent({ event: 'caregivers-10-10cg--submission-successful' });
```

Comment:
I am trying to wait for a build to see if I need this code, I don't think I need it, but I don't see a build on this PR -_- Maybe opening it up for review will make it build?